ext/zend_test/test.c

```diff
@@ -535,6 +536,11 @@ static ZEND_METHOD(ZendTestNS_Foo, method)
 	RETURN_LONG(0);
 }
 
+static ZEND_METHOD(ZendTestNS_UnlikelyCompileError, method)
+{
+	ZEND_PARSE_PARAMETERS_NONE();
```

Comment:
Since this method has a `UnlikelyCompileError` return type, it needs to return an instance of `UnlikelyCompileError`.